<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IServerEditor</class>
 <widget class="QDialog" name="IServerEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit servers</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/toolicons/gfx/options.png</normaloff>:/toolicons/gfx/options.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>197</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="btnNew">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/gfx/new.png</normaloff>:/icons/gfx/new.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3" colspan="3">
    <widget class="QTreeView" name="serverView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLineEdit" name="edPassword">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="edName"/>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="edServer"/>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="btnDelete">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/gfx/remove.png</normaloff>:/icons/gfx/remove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QToolButton" name="btnSave">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/gfx/save.png</normaloff>:/icons/gfx/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QSpinBox" name="edPort">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>6667</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionNewNetwork">
   <property name="text">
    <string>New network</string>
   </property>
  </action>
  <action name="actionNewServerNetwork">
   <property name="text">
    <string>Assign to selected network</string>
   </property>
  </action>
  <action name="actionNewServerNoNetwork">
   <property name="text">
    <string>Not assigned with a network</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
